docs/source/science/ATL03/ATL03.ipynb

```diff
@@ -2,13 +2,57 @@
  "cells": [
```

Comment:
<p>Can you check the concept ID of the collection in which the data is being pulled from? I see that there's two different listings with two different concept IDs in EarthData Search, one from the ?NSIDC server and one on EarthData Cloud</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Looks like it's using the one from EarthData Cloud

Reply:
If _where_ the data is being pulled from matters, should we change the wording then? Or that portion could be removed?

Reply:
I'll probably just go ahead and remove that portion, I don't think it really has to be there